src/components/frame/v5/pages/StreamingPaymentsPage/partials/StreamingPaymentsTable/StreamingPaymentsTable.tsx

```diff
@@ -0,0 +1,113 @@
+import { BigNumber } from 'ethers';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { DEFAULT_TOKEN_DECIMALS } from '~constants';
+import { useGetStreamingPaymentsQuery } from '~gql';
+import { formatText } from '~utils/intl.ts';
+import { getAmountPerValue } from '~utils/streamingPayments.ts';
+import Table from '~v5/common/Table/Table.tsx';
+
+import { useRenderSubComponent, useStreamingTableColumns } from './hooks.tsx';
+import { type StreamingTableFieldModel } from './types.ts';
+
+const displayName =
+  'pages.StreamingPaymentsPage.partials.StreamingPaymentsTable';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Streaming Payments',
+  },
+});
+
+const StreamingPaymentsTable = () => {
+  const { data, loading } = useGetStreamingPaymentsQuery();
+  const streamingPayments: StreamingTableFieldModel[] = (
+    data?.searchStreamingPayments?.items || []
+  ).reduce<StreamingTableFieldModel[]>((result, item) => {
+    if (!item) {
+      return result;
+    }
+
+    const { recipientAddress } = item;
+
+    let existingEntryIndex = result.findIndex((entry) => {
+      return entry.user === recipientAddress;
+    });
+
+    const actionData = {
+      title: item.actions?.items[0]?.metadata?.customTitle || '',
+      token: {
+        symbol: item.token?.symbol || '',
+        address: item.tokenAddress,
+        decimals: item.token?.decimals || DEFAULT_TOKEN_DECIMALS,
+      },
+      team: item.nativeDomainId,
+      paymentId: item.id,
+      streamed: item.amount,
+      initiator: item.creatorAddress,
+      recipient: item.recipientAddress,
+      period: getAmountPerValue(item.interval).toLowerCase(),
+      transactionId: item.actions?.items[0]?.transactionHash || '',
+    };
+
+    if (existingEntryIndex < 0) {
+      result.push({
+        user: recipientAddress,
+        userStreams: {
+          amount: '0',
+          tokenAddress: item.tokenAddress,
+          tokenDecimals: item.token?.decimals || DEFAULT_TOKEN_DECIMALS,
+        },
+        actions: [],
+      });
+
+      existingEntryIndex = result.length - 1;
+    }
+
+    const totalAmount = BigNumber.from(
+      result[existingEntryIndex].userStreams.amount,
+    )
+      .add(BigNumber.from(item.amount))
+      .toString();
+
+    const actions = [...result[existingEntryIndex].actions, actionData];
+
+    const updatedResult = [...result];
+    updatedResult[existingEntryIndex] = {
+      ...updatedResult[existingEntryIndex],
+      actions,
+      userStreams: {
+        amount: totalAmount,
+        tokenAddress: item.tokenAddress,
+        tokenDecimals: item.token?.decimals || DEFAULT_TOKEN_DECIMALS,
+      },
```

Comment:
I think this will add the amounts altogether no matter the token, and only persist the last token address